source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -7642,16 +7436,45 @@ export class SerializationContext extends BaseSerializationContext {
         return result;
     }
 
+    parseSingletonElement(parent: CardElement | undefined,
+        source: any,
+        allowFallback: boolean): CardElement | undefined {
+
+        return this.parseCardObject<CardElement>(
+            parent,
+            source,
+            [],
+            allowFallback,
+            (typeName: string) => {
+                return this.singletonElementRegistry.createInstance(typeName, this.targetVersion);
+            },
+            (typeName: string, errorType: Enums.TypeErrorType) => {
+                if (errorType === Enums.TypeErrorType.UnknownType) {
+                    this.logParseEvent(
+                        undefined,
+                        Enums.ValidationEvent.UnknownElementType,
+                        Strings.errors.unknownElementType(typeName));
+                }
+                else {
+                    this.logParseEvent(
+                        undefined,
+                        Enums.ValidationEvent.ElementTypeNotAllowed,
+                        Strings.errors.elementTypeNotAllowed(typeName));
+                }
+            });
+    }
+
+
     parseElement(
         parent: CardElement | undefined,
         source: any,
         allowFallback: boolean): CardElement | undefined {
 
         let forbiddenTypes: string[] = [];
-        if (parent?.isInCarouselCard())
-        {
-            forbiddenTypes = Array.from(GlobalRegistry.forbiddenCarouselElements.values());
-        }
+        // if (parent?.isInCarouselCard())
+        // {
+        //     forbiddenTypes = Array.from(GlobalRegistry.forbiddenCarouselElements.values());
+        // }
```

Comment:
why is this commented?

Reply:
as noted in the PR description, it's not implemented here yet...